core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/footer.properties

```diff
@@ -1 +1,2 @@
 tooltip=There are {0} active administrative monitors.
+tooltipSec=There are {0} active security administrative monitors.
```

Comment:
thoughts on?
```suggestion
tooltipSec={0} active security issue(s).
```

Reply:
Seems misleading.

Reply:
Less information and open to interpretation